easybuild/easyconfigs/g/GULP/GULP-6.1-foss-2021b.eb

```diff
@@ -0,0 +1,63 @@
+# Contribution from Imperial College London / UK 
+# Based on GULP-6.0-foss-2020b.eb 
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'GULP'
+version = '6.1'
+
+homepage = 'https://gulp.curtin.edu.au/gulp/'
+description = """GULP is a program for performing a variety of types of simulation on materials
+ using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs
+ and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations"""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+patches = [
+    'GULP-6.1_fftw-blas-linking-gcc-11.patch',
+]
+checksums = [
+    '503dff253f4686c570adfd3ff75a657e29151b4255dec726daa50eda5a7ac3f4',  # gulp-6.1.tgz
```

Comment:
This checksum is no longer correct for fresh downloads, see #15984 for more details...

Reply:
Thanks for letting me know. Do you want me to updated the EC file for the new checksum?

Reply:
I think we'll need to go with skipping the checksum verification for GULP (by using `None` as checksum), since it keeps changing...

Reply:
I am trying to reach out to one of the developers. Would it be ok to keep that for now until 16.8. say? I want to avoid too many changes.